### PR TITLE
Api point for KP

### PR DESCRIPTION
**Pull request by sumit-158**
### What
- Creating Api point for KP

### Fixes bug(s)
- #2

### CODE REVIEW

**Comment by sumit-158** [comment]
@alexgarel can you please look into the error :-) , what I'm doing wrong
image

File: app/main.py [hunk 1]
```diff
@@ -27,3 +27,7 @@ def ansewer_questions_brand_president():
             ],
         }
 
+@app.get("/{facet_name}/{facet_value}")
+def knowledge_panel(facet_name :Facetname,facet_value: Facetvalue ):
```
Comment (alexgarel) [inline app/main.py:31]: why not just use a str for facet_value ?

File: app/models.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
+from pydantic import BaseModel
+from enum import Enum
+
+class Facetname(str, Enum):
```
Comment (alexgarel) [inline app/models.py:4]: Facetname should be in singular for how we will use them.

https://world.openfoodfacts.org/countries/ for all countries

But **country** for a single country : https://world.openfoodfacts.org/country/india
(and your urls are for single values)

File: app/main.py (hunk 1 above)
Comment (sumit-158) [reply app/main.py:31]: it is a str, U mean for facetname ?

**Review by alexgarel — APPROVED** [review]
LGTM

**Pull request merged**